### ISSUE TITLE
Splitting the Race Patcher

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#494]
Just formalizing what I've been planning to do for a while now. Sub-issue of #312.

## TL;DR
The Race patcher is far too complex for a single patcher. It's more complex than Import Actors, and that's split into *four* patchers!
Splitting it is necessary if we want to have any chance of moving it to newer games.

## Details
My original plan was to split it into three patchers - `Import Races`, `Tweak Races` and `Race Fixer`/`NPC Fixer` (not sure about that name).
However, my attempt at splitting it failed miserably, because I didn't properly understand how it works.

It turns out the CBash version *is already* nicely split into five patchers, they've just been hackily glued together:
 - `Import Races` - simple preserving importer, handles `Body-F`, `Body-M`, `Body-Size-F`, `Body-Size-M`, `Hair`, `R.Attributes-F`, `R.Attributes-M`, `R.Description`, `R.Ears`, `R.Head`, `R.Mouth`, `R.Skills`, `R.Teeth`, `Voice-F` and `Voice-M`
 - `Import Races: Eyes` - this *still* does too much, it merges eyes via the `Eyes` tag but also does all the crazy eye/hair fixing
 - `Import Races: Relations` - merger/preserver, handles `R.Relations`, very similar to Import Relations, maybe we could invent an `R.Derel` tag too
 - `Import Races: Spells` - merger/preserver, handles `R.AddSpells` and `R.ChangeSpells`
 - `Tweak Races` - houses the races tweaks (e.g. Bigger Nords and Orcs)

Notably absent is the NPC fixing functionality of the PBash version, CBash just doesn't seem to have that.

Final split design, arrived at after one attempt that failed due to regressions:
 - `Import Races` - same as above
 - `Import Races: Relations` - same as above
 - `Import Races: Spells` - same as above
 - `Tweak Races` - same as above
 - `Eye Checker` - the googly eye filtering
 - `Race Checker` - the hair/eye sorting
 - `NPC Checker` - the NPC eye/hair fixing

**Comment by Utumno** [comment]
From my digging in there - the `CBash_ActorImporter`, `CBash_CellImporter` and the `_CBashOnlyRacePatchers` are the reason the `autoKey` gui patchers attribute leaks in the model _and_, therefore, the main reason for the bashTags argument in scan() (not the only reason, unfortunately). We could pass different sets of sources based on the bashTags - but this does not seem very elegant either.
